picard/formats/apev2.py

```diff
@@ -204,8 +204,22 @@ def _get_tag_name(self, name):
         else:
             return name.title()
 
-    def supports_tag(self, name):
-        return bool(name)
+    @classmethod
+    def supports_tag(cls, name):
+        unsupported_tags = {
+            'podcasturl',
+            'gapless',
+            'showsort',
+            'show',
+            'arranger',
+            'musicip_fingerprint',
+            'podcast',
+            'totaltracks'}
+        return (bool(name)
+                and name not in unsupported_tags
+                and not name.startswith('performer')
+                and not name.startswith('comment')
```

Comment:
Shouldn't we use `name.startswith('performer:') and not name.startswith('comment:')` instead ?